components/ActivityMap/drawActivities/drawActivities.ts

```
@@ -0,0 +1,241 @@
+import { getHeatmapColorForCount } from './utils/getHeatmapColorForCount';
+import { GPXTrack } from "@/lib/types/types";
+import L from 'leaflet';
+import logger from "@/lib/utils/logger";
+
+interface HeatmapCache {
+    imageUrl: string;
+    bounds: any;
+    zoomLevel: number;
+}
+
+let heatmapCache: HeatmapCache | null = null;
```

ESLint

[error] 'heatmapCache' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)